pkg/operator/starter.go

```diff
@@ -176,6 +177,7 @@ var deploymentConfigMaps = []revision.RevisionResource{
 	{Name: "config"},
 	{Name: "controller-manager-kubeconfig"},
 	{Name: "serviceaccount-ca"},
+	{Name: "service-ca"},
```

Comment:
This might need to be set with Optional: true since it's not an installer created resource

Reply:
I thought so too: https://github.com/openshift/cluster-kube-controller-manager-operator/pull/178#discussion_r259874110